_posts/2020-02-12-myft-machine-learning.html

```diff
@@ -0,0 +1,105 @@
+
+<p>So Suggestr was born:  <a href="https://github.com/ftlabs/suggestr">https://github.com/ftlabs/suggestr</a></p>
+
+{% include figure.html src="/assets/uploads/2020/02/myft_ml_suggestr_output.png" description="Example screenshot of Suggestr's output when providing Blockchain as an input topic. Core results are in the results object, the rest of the output shows the steps taken to get the results, as not to create too much of a black box recommendation." width="80%" %}
+
+<h2>Suggestr's mission</h2>
+
+<p>Using topic cluster analysis, topic correlations data and topic follow/unfollow ratios - can we suggest retrieve topics to follow (or retrieve articles) for based on any given topic(s)?</p>
+
+<p>The first prototype of Suggestr works well but we'd like to refine and add to the ranking filters before user (reader) testing can begin.</p>
+
+<h2>What did we learn?</h2>
+
+<p>All those ML & AI experts saying data prep is the largest time consumption percentage of the project weren't kidding. However all that time familiarised us with the data and allowed us to build effective tools to minimise time spent if we revisit that data.</p>
+
+<p>If we don't record the data we have in a documented or understandable way that priceless data is basically number garbage. You need to be able to trust your data, to trust your analysis for your users to trust your output. So let your use of the recorded data shape the future recorded data, a cycle that will continue to improve your data (and projects) into the future.</p>
+
+<p>
+<i>"Every company has messy data, and even the best AI companies are not fully satisfied with their data."</i>
+<br />
+<b>Andrew Ng</b>
+</p>
+
+<p>It is difficult to generate recommendations based on a few metrics, you'll want as many as you can to shape the best suggestion. In the end there may be no good suggestions right now, and it's okay to say that rather than making a bad/false suggestion and losing the trust of a user.</p>
+
+<p>Our bumbling around in the data helped other teams understand how tricky navigating this data can be without help. Efforts are already underway to clarify and clean the data for everyone's use.</p>
+
+<h2>What does the future hold?</h2>
+
+<p>While currently on hold we look forward to completing Suggestr in the future and testing the quality of its recommendations against a live audience. There's an interest here at the FT to find recommendation model(s) that fit each user best, or that users can pick and choose from.</p>
```

Comment:
I would remove the brackets around `(s)`, otherwise there's an "a" missing before recommendation.

Reply:
Done